tests/plugins/tableselection/manual/mouseover.md

```diff
@@ -0,0 +1,20 @@
+@bender-ui: collapsed
+@bender-tags: tc, bug, 4.7.1, 515
+@bender-ckeditor-plugins: wysiwygarea, toolbar, sourcearea, tableselection
+
+**Procedure:**
+
+1. Open developer console.
+2. Focus on editor.
+3. Scroll down/up.
```

Comment:
Can 3 and 4 be combined? Write that the editor should be scrolled using a mouse, so this will automatically force mousing over.